tools/build_defs/framework.bzl

```diff
@@ -180,6 +180,7 @@ def cc_external_rule_impl(ctx, attrs):
     ]
 
     script_text = "\n".join(script_lines)
+    print("script text: " + script_text)
```

Comment:
Is this intentional print?

Reply:
Yep; I added this in another pull request already.